### ISSUE TITLE
Mailbox - Link missing

### ISSUE THREAD

**Issue by thogebati** [amosproj/amos2023ws01-ticket-chat-ai#15]
## User story
1. As a user that wants to report a problem/incident or wants to create a service request via email
2. I want / need mailbox that I can send mails to
3. So that a ticket will be created filled with all the information from my email

## Acceptance criteria
* The mail actually reaches the mailbox
* I get an email notification that my mail was received.
* That a ticket is created (link issue here)

## Definition of done (DoD)
* Added only after week 5
* The same for all features
* Here goes the project specific part

## DoD general criteria
* Feature has been fully implemented
* Feature has been merged into the mainline
* All acceptance criteria were met
* Product owner approved features
* All tests are passing
* Developers agreed to release
